### PR TITLE
Added "after-offset" filter UI - closes #14

### LINKED ISSUE #14: Filter data with a starting offset
Already available on url querystring with ?after=partition-offset, simply need an ui

### PR DESCRIPTION
**Pull request by zthulj**
Hi,

This is my work in order to close the #14.

I simply add a filter called "After offset" in the UI and a partitionNB field from the controller to be able to generate the form. 

Please let me know if you want me to adjust anything.

### CODE REVIEW

File: src/main/resources/views/blocks/topic/data.ftl [hunk 1]
```diff
@@ -105,6 +106,42 @@
...
                 </div>
             </li>
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle"
+                   href="#"
+                   role="button"
+                   data-toggle="dropdown"
+                   aria-haspopup="true"
+                   aria-expanded="false">
+                    <strong>After offset:</strong> ()
```
Comment (tchiotludo) [inline src/main/resources/views/blocks/topic/data.ftl]: Change to After only, it break responsive design 

image

File: src/main/resources/views/blocks/topic/data.ftl [hunk 2]
```diff
@@ -105,6 +106,42 @@
...
+                                            <#if navbar["offset"][partition?string]??>
+                                                value="${navbar["offset"][partition?string]}"
+                                            </#if>
+                                        />
+                                    </td>
+                                </tr>
+                            </#list>
+
+                        </table>
+                        <div class="input-group-append">
```
Comment (tchiotludo) [inline src/main/resources/views/blocks/topic/data.ftl]: This button is missing 1 breakpoint : 
image

File: src/main/resources/views/blocks/topic/data.ftl (hunk 1 above)
Comment (tchiotludo) [inline src/main/resources/views/blocks/topic/data.ftl]: remove `()` not needed

**Comment by zthulj** [comment]
My bad, I just learned that Kafkahq is responsive, never tested it before :satisfied: . Will take care for next PR.

The flex display was working differently in medium width cause the partitions table wasn't taking 100%. It's ok now.

**Comment by tchiotludo** [comment]
Yes !! Responsive is a minimum also for an admin tools nowadays, and easy with bootstrap :smiley: 
Good stuff here ! Thanks for the work

**Pull request merged**